README.rst

```diff
@@ -43,11 +43,11 @@ Why would I use this instead of Django's `manage.py test` command?
 Running your test suite with pytest-django allows you to tap into the features
 that are already present in pytest. Here are some advantages:
 
-* `Manage test dependencies with pytest fixtures. <http://pytest.org/latest/fixture.html>`_
+* `Manage test dependencies with pytest fixtures. <http://pytest.org/en/latest/fixture.html>`_
 * Less boilerplate tests: no need to import unittest, create a subclass with methods. Write tests as regular functions.
```

Comment:
Woah!
Looks more like an issue with the redirection from readthedocs (if we're using this?!): it used to select the user's language etc..  using `https` (where it redirects to) does not help though.